docs/tutorials/mastering.md

```diff
@@ -1,6 +1,9 @@
 # Mastering
 
-In this tutorial, we will deploy and run the SD-Core 5G core network following Control and User Plane Separation (CUPS) principles. The radio and cell phone simulator will also be deployed on an isolated cluster. [Multipass](https://multipass.run/) is used to create separate VMs connected with [LXD](https://ubuntu.com/lxd) networking.
+In this tutorial, we will deploy and run the SD-Core 5G core network following Control
+and User Plane Separation (CUPS) principles. The radio and cell phone simulator will also
```

Comment:
I don't think it's a good idea to add line breaks to all the longer lines, just use word wrap.  What ends up happening is if words are added to the middle of a line, then you have to manually word wrap all over the place.

Reply:
True. Reverted.